gulp-tasks/reference-docs/build-reference-docs.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const path = require('path');
+const os = require('os');
+const fs = require('fs-extra');
+
+const getLatestTags = require('./get-latest-tags');
+const filterTagsToBuild = require('./filter-tags-to-build');
+const getSourceCode = require('./get-source-code');
+const buildJSDocs = require('./build-js-docs');
+
+const buildReferenceDocs = (gitUrl, docPath, jsdocConfPath) => {
```

Comment:
Please document functions with JSDoc. Include at least a brief description, and `@param`/`@return` as appropriate.  Will still merge if not added.